### DISCUSSION TITLE
请问充值功能

### DISCUSSION THREAD

**Discussion by lishan256** [ZuodaoTech/everyone-can-use-english#677] [Ideas]
请问付款能够直接使用美元结算吗？我看充值都是先要转为日元再转成美元。如果有美元卡，好像也只能先转成相应的日元，再转美元，不太明白为什么不能直接刷美元。

**Comment by an-lee** [comment]
因为美元的收款渠道还没弄好，暂时只有日元的渠道可用。后面弄好了会改过来。

**Reply by lishan256** [reply]
多谢回复！期待enjoy越来越完善！也非常感谢您的付出，让enjoy越来越好啦！！
